packages/cli/package.json

```diff
@@ -7,8 +7,12 @@
   },
   "bugs": "https://github.com/stoplightio/prism/issues",
   "dependencies": {
+    "@stoplight/http-spec": "^1.0.4",
+    "@stoplight/json-ref-resolver": "^2.0.3",
     "@stoplight/prism-core": "^3.0.3",
     "@stoplight/prism-http-server": "^3.0.3",
+    "@stoplight/yaml": "^2.8.0",
+    "axios": "^0.18.0",
```

Comment:
Seems like we're removing node-fetch and adding axios, but last time we talked we were going to remove axios and keep node-fetch? Was that a miscommunication or were there some new reasons?

Reply:
Ah, I thought this was going to be a separate PR — do you want to couple the node-fetch replacement here?

Wouldn't it be better to have a dedicated PR?

Reply:
Ok if you make a story for it I’ll approve this.